BUILDING.md

```diff
@@ -80,22 +80,9 @@ for details.
 
 ### Update vendored dependencies
 
-To update dependency you need just change `vendor.conf` file and run `vndr` tool:
+To update dependency you need just change `vendor.mod` file and run `go mod vendor` tool:
```

Comment:
Should this be `make go-mod-vendor` ? (nitpicking; perhaps we should name the Makefile target `vendor`, but "meh")